other/requirements.md

```diff
@@ -0,0 +1,57 @@
+# SMF Miniumn Requirements
+
+## PHP
+### PHP Version Support
+| MIN SMF VERSION | MAX SMF VERSION | MIN PHP VERSION | MAX PHP VERSION |
+| ------ | ------ | ------ | ------ |
+| 3.0    | LATEST | 8.0.0  | LATEST  |
```

Comment:
Should we also include 2.1 here? Thoughts?
```suggestion
| 2.1    | LATEST | 7.0.0  | LATEST  |
| 3.0    | LATEST | 8.0.0  | LATEST  |
```

Reply:
We can, but the 2.1 branch can also hold it.

Reply:
Are the branches mentioned somewhere? This may cause confusion since different projects tend to do one repo per version thus making it non-trivial unless you previously contributed to 2.1 or you check the branches.